src/python/pants/option/global_options.py

```diff
@@ -96,12 +96,11 @@ def register_bootstrap_options(cls, register):
                   'subprocesses that outlive the workdir data (e.g. `./pants server`).')
     register('--pants-config-files', advanced=True, type=list,
```

Comment:
Erm, this is broken and a can of worms, filed #4717, may revert the `-c` in the short term.